lib/msf/core/db_manager/import/qualys/scan.rb

```diff
@@ -5,22 +5,23 @@ def import_qualys_scan_xml(args={}, &block)
     bl = validate_ips(args[:blacklist]) ? args[:blacklist].split : []
 
 
-    doc = rexmlify(data)
-    doc.elements.each('/SCAN/IP') do |host|
+    doc = Nokogiri.XML(data)
+    doc.xpath('/SCAN/IP').each do |host|
       hobj = nil
-      addr  = host.attributes['value']
+      addr  = host.attr('value')
```

Comment:
Should we not be checking if this is `nil` prior to `yeild`ing it?

Reply:
Limiting change here seems appropriate, the previous pattern passed `nil` if the attribute was not found.